src/qml/qgismobileapp.qml

```diff
@@ -1352,21 +1352,21 @@ ApplicationWindow {
     Connections {
         target: iface
 
-        onLoadProjectStarted: {
+        function onLoadProjectStarted() {
           messageLogModel.suppressTags(["WFS","WMS"])
         }
     }
 
     Connections {
       target: qfieldAuthRequestHandler
 
-      onShowLoginDialog: {
+      function onShowLoginDialog() {
```

Comment:
Not sure at all, but doesn't it need here an argument like `onShowLoginDialog( realm )`?

Just picked up this as an example - I haven't checked the other connections...

Reply:
Correct, this is required (wfs login dialog does not show up otherwise)